airflow/sensors/external_task.py

```diff
@@ -143,18 +143,27 @@ def __init__(
         if external_task_id is not None and external_task_ids is not None:
             raise ValueError(
                 "Only one of `external_task_id` or `external_task_ids` may "
-                "be provided to ExternalTaskSensor; not both."
+                "be provided to ExternalTaskSensor; "
+                "Use external_task_id or external_task_ids or external_task_group_id."
```

Comment:
```suggestion
                "be provided to ExternalTaskSensor; "
                "use external_task_id or external_task_ids or external_task_group_id."
```

Same for other messages below. Or you could use a period instead.

Reply:
Done the change.